grafana_collector/grafana/dashboard.go

```diff
@@ -31,33 +31,61 @@ package grafana
 
 import (
 	"encoding/json"
-	"github.com/ngaut/log"
-	"net/url"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"regexp"
+	"sort"
 	"strings"
+	"time"
+
+	"github.com/ngaut/log"
 )
 
+// ScopedVar represents template variable
+type ScopedVar struct {
+	Text  string
+	Value string
+}
+
 // Panel represents a Grafana dashboard panel
 type Panel struct {
-	ID    int
-	Type  string // Panel Type: Graph/Singlestat
-	Title string
+	ID         int
+	Type       string // Panel Type: Graph/Singlestat
+	Title      string
+	RowTitle   string
+	ScopedVars map[string]ScopedVar
 }
 
 // Row represents a container for Panels
 type Row struct {
-	ID        int
-	Showtitle bool // Row is visible or hidden
-	Title     string
-	Panels    []Panel
+	ID              int
+	Showtitle       bool // Row is visible or hidden
+	Title           string
+	Repeat          string
```

Comment:
Why not to extract `repeatxx` as a structure/object

Reply:
We need it to unmarshal the dashbaord  API JSON, so keep the same structure.